docs/src/markdown/installation.md

```diff
@@ -10,22 +10,22 @@ The recommended way to install RegReplace is via [Package Control][package-contr
 
 3. Start typing `RegReplace`; when you see it, select it.
 
-4. Restart to be sure everything is loaded proper.
+4. Restart to be sure everything is loaded properly.
 
 5. Enjoy!
 
 ## Manual Installation
 
 !!! warning "Warning"
-    This is not the recommended way to install RegReplace for the average user.  Installing this way **will not** get automatically updated.
+    This is not the recommended way to install RegReplace for the average user.  Installing it this way **will not** get automatically updated.
 
     If you are forking for a pull request, you should **just** clone RegReplace and run Package Control's `Satisfy Dependency` command to get all the dependencies.
 
-For those who want to install RegReplace without package control, here are the steps.  It understood that some people for what ever reason will prefer manually and may even have legitimate reasons to do so.  When going this route, you will have to keep all the packages updated yourself.
+For those who want to install RegReplace without package control, here are the steps.  Some people prefer manual installation — and may have legitimate reasons to do so.  When going this route, you will have to keep all the packages updated yourself.
```

Comment:
General note: I usually prefer using `--` which smarty will convert to the Unicode dash.  I often can't tell which is being is used `—` or `-` when looking at documents with certain fonts.

Reply:
To be honest, I'm not sure an emdash makes sense here being followed by `and`.  I think the emdash can be removed altogether.

Reply:
I actually think we can remove this sentence: `Some people prefer manual installation — and may have legitimate reasons to do so.`

Reply:
I agree with removing the sentence, it doesn't really contribute much — possibly, users choosing manual installation are already expert enough to know what they are doing and why; unexperienced users, on the other hand, probably won't care about the reasons why others do what's beyond their understanding.

As for the dashes, I'm just so used to manually enter them as chars that I do it automatically. I think it's better using the actual char because some MD converters/previews don't deal with smart punctuation (for example here, in GH issues, the "--" and "---" aren't converted. Technically, markdown docs should contain stricly ASCII chars only, but in practice everyone uses UTF8 freely in markdown docs, and very few parsers will object to it.

Reply:
> agree with removing the sentence, it doesn't really contribute much — possibly, users choosing manual installation are already expert enough to know what they are doing and why; unexperienced users, on the other hand, probably won't care about the reasons why others do what's beyond their understanding.

I sometimes write what I'm thinking even if it doesn't contribute much.  It's only times like this when I'm reviewing it that I realize it was kind of a pointless statement.

> I think it's better using the actual char because some MD converters/previews don't deal with smart punctuation (for example here, in GH issues, the "--" and "---" aren't converted. 

I'll be honest, I'm not concerned with what other parsers would do with *these* docs.  I try to explain up front that I expect them to be rendered in the mkdocs environment with the specified toolchain.  I do have a biased self interest in the way I format my docs.  I even use custom extensions to assist in doing custom things. I'm not against Unicode in Markdown, but as stated in my documents, emdash and `-` are difficult to distinguish with certain fonts.  The toolchain is easy to install, `pip install mkdocs-material` will actually install all the requirements. And with `virtualenv` you can even leave your default Python environment untouched.

Even GitHub does its own thing as GFM is *not* Markdown, but a Markdown-ish derivative.  Though they may be going down a more CommonMark path now, but even that is Markdown-ish as well, even if it is trying to establish itself as the One True Markdown.